pyproject.toml

```diff
@@ -40,3 +40,34 @@ skip = [".gitignore", ".dockerignore"]
 
 [tool.setuptools_scm]
 local_scheme = "dirty-tag"
+
+
+[tool.pytest.ini_options]
+minversion = 6.0
+python_files = "test_*.py"
+python_functions = "test_*"
+junit_family = "xunit2"
+junit_logging = "system-out"
+console_output_style = "classic"
+log_level = "ERROR"
+log_cli = "true"
+filterwarnings = ["ignore::DeprecationWarning"]
+addopts = "-rfE --color=yes -p no:ddtrace -p no:ddtrace.pytest_bdd --cov=sheepdog --cov-report term"
```

Comment:
You can remove the filter warnings and also --cov seems to be pointing to the wrong module

Reply:
removed and fixed.